meta/internal/meta.pb.go

```diff
@@ -518,8 +525,8 @@ func (m *ShardOwner) GetNodeID() uint64 {
 }
 
 type ContinuousQueryInfo struct {
-	Name             *string `protobuf:"bytes,1,req" json:"Name,omitempty"`
-	Query            *string `protobuf:"bytes,2,req" json:"Query,omitempty"`
+	Name             *string `protobuf:"bytes,1,req,name=Name" json:"Name,omitempty"`
```

Comment:
These changes aren't even made by me, they are made by whatever generates it.  This is an issue with vendor management.  Meaning that whoever last generated didn't have the same version I have.  Not sure how to fix this until we adopt a vendor versioning approach.  This happens pretty much any time someone generates this.